comms/dht/src/dht.rs

```diff
@@ -46,13 +49,22 @@ use tari_comms::{
     pipeline::PipelineError,
 };
 use tari_shutdown::ShutdownSignal;
-use tokio::task;
 use tower::{layer::Layer, Service, ServiceBuilder};
 
+const LOG_TARGET: &str = "comms::dht";
+
 const DHT_ACTOR_CHANNEL_SIZE: usize = 100;
 const DHT_DISCOVERY_CHANNEL_SIZE: usize = 100;
 const DHT_SAF_SERVICE_CHANNEL_SIZE: usize = 100;
 
+#[derive(Debug, Error)]
```

Comment:
You should be using `thiserror`

Reply:
I'd rather migrate the whole crate over at once